osh/hub/scan/notify.py

```diff
@@ -56,8 +56,7 @@ def get_recipient(user):
     if user.email:
         return user.email
 
-    # XXX: hardcoded
-    return user.username + "@redhat.com"
+    return "@".join([user.username, settings.DEFAULT_EMAIL_DOMAIN])
```

Comment:
I find this unnecessarily cryptic.  Could we please write this as `user.username + "@" + settings.DEFAULT_EMAIL_DOMAIN` or `f"{user.username}@{settings.DEFAULT_EMAIL_DOMAIN}"`?